package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "eslint-config-incuna",
+  "version": "1.0.0",
+  "description": "Incuna eslint shared config",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Could implement a test that simply requires the main file, to cover syntax errors:
```json
"scripts": {
    "test": "node index.js"
}
```